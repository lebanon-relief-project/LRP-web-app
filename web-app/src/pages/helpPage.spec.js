import React from "react";
import { render } from "@testing-library/react";
import HelpPage from "./helpPage";
import "jest-styled-components";

jest.mock("../components/FlashCardsSection", (props) => {
  return (props) => <div>Flash cards section</div>;
});

describe("the HelpPage component", () => {
  it("should match the snapshot", () => {
    const { container } = render(<HelpPage />);
    expect(container).toMatchSnapshot();
  });
});
