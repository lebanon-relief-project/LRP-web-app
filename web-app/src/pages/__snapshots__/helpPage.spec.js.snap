// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the HelpPage component should match the snapshot 1`] = `
.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  z-index: 2;
  background-color: transparent;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  position: absolute;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  height: 363px;
  width: 100%;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: transparent;
  margin-left: 16.7%;
  max-width: 632px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  z-index: 1;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background: #002766;
  margin: 0;
  margin-top: 64px;
  padding: 0;
  width: 100%;
  height: 363px;
  z-index: 0;
}

.c7 {
  font-weight: bold;
  font-size: 38px;
  margin: 0;
  color: #ffffff;
}

.c9 {
  margin: 0;
  margin-bottom: 1rem;
  font-size: 16px;
  color: #ffffff;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.c11 {
  width: 60%;
  padding-top: 2rem;
  padding-bottom: 2rem;
  margin: 0 16.7%;
  background: inherit;
}

.c10 {
  background: #ffffff;
  margin: 0;
  padding: 0;
  width: 100%;
}

.c12 {
  font-weight: bold;
  font-size: 24px;
}

.c14 {
  margin: 0;
  font-size: 16px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media (max-width:576px) {
  .c8 {
    margin-top: 15px;
  }
}

@media (max-width:576px) {
  .c5 {
    height: 542px;
  }
}

@media (max-width:768px) {
  .c6 {
    margin-left: 30px;
  }
}

@media (max-width:576px) {
  .c6 {
    margin-left: 18px;
    margin-right: 18px;
    -webkit-flex: 1;
    -ms-flex: 1;
    flex: 1;
    margin-top: 20px;
  }
}

@media (max-width:1025px) {
  .c3 {
    display: none;
  }
}

@media (max-width:1025px) {
  .c4 {
    display: none;
  }
}

@media (max-width:576px) {
  .c1 {
    height: 542px;
  }
}

@media (max-width:576px) {
  .c7 {
    font-size: 30px;
    line-height: 38px;
  }
}

@media (max-width:1025px) {
  .c11 {
    width: auto;
    width: 75%;
  }
}

@media (max-width:768px) {
  .c11 {
    margin: 0 30px;
    width: 90%;
  }
}

@media (max-width:576px) {
  .c11 {
    margin: 25px 18px;
    padding-top: 0px;
    padding-bottom: 0px;
    width: auto;
  }
}

<div>
  <div
    class="c0"
  >
    <div>
      <div
        class="c1"
      >
        <div
          class="c2"
        >
          <div
            class="c3"
          >
            <img
              alt="background"
              src="HelpPageLeftBackground.svg"
            />
          </div>
          <div
            class="c4"
          >
            <img
              alt="background"
              src="HelpPageRightBackground.svg"
            />
          </div>
        </div>
        <div
          class="c5"
        >
          <div
            class="c6"
          >
            <h2
              class="c7"
            >
              Traumatic Events
            </h2>
            <div
              class="c8"
            >
              <p
                class="c9"
              >
                Trauma derives from an exposure to an event resulting in changes of feelings, thoughts or behaviors. These emotional and physical changes accompanying the symptoms of a traumatic experience constitute a normal reaction to an abnormal event.
              </p>
              <p
                class="c9"
              >
                Trauma is unique, and any distressing event can result in it. It can be widespread, and it can be personal, varying from pandemics, to economic crises, death and abuse.
              </p>
              <p
                class="c9"
              >
                Trauma can also be experienced by watching or helping someone who has been affected by a traumatic event. Each person lives trauma differently, and symptoms or modifications in thoughts, behaviors and emotions vary from one person to another.
              </p>
            </div>
          </div>
        </div>
      </div>
      <section
        class="c10"
      >
        <div
          class="c11"
        >
          <h2
            class="c12"
          >
            Something feels different
          </h2>
          <div
            class="c13"
          >
            <p
              class="c14"
            >
              If you feel that you are experiencing emotional and physical changes since a distressing event, know that you are not alone, and that it is normal to wanting to understand your changes after a traumatic event. The cards below give you a brief description of what you may be experiencing. Please read them and select those that apply to you. Once you are done, and want to know more about why you are experiencing those feelings and how to cope with them, click on give me advice at the bottom of the page.
            </p>
          </div>
        </div>
      </section>
      <div>
        Flash cards section
      </div>
    </div>
  </div>
</div>
`;
